### PR TITLE
Tiger technique

### PR DESCRIPTION
**Pull request by ScalarVector1**
###What are the specifics of what is being added?
The tiger technique accessory, which allows the execution of custom combos with axes.

###Is there an associated suggestion or dev server #code-needed post? How well does this PR follow it?
https://trello.com/c/uqrAW6ek/118-tiger-technique-5

###Is there anything left to be done in this addition?
Obtainment methods to be done as a separate ticket.

### CODE REVIEW

File: Content/Items/Misc/Accessories.AxeBook.cs [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+
+namespace StarlightRiver.Content.Items.Misc
+{
+	class AxeBook : SmartAccessory
+	{
+		public int comboState;
+
+		public AxeBook() : base("Tiger Technique", "Allows execution of combos with axes\nThe final strike will rend enemy armor\nRight click to throw your axe") { }
+
+		public override string Texture => AssetDirectory.MiscItem + "AxeBook";
```
Comment (ScalarVector1) [inline Content/Items/Misc/Accessories.AxeBook.cs:24]: property override should be before ctor

File: Content/Items/Misc/Accessories.AxeBook.cs [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+			return false;
+		}
+
+		/// <summary>
+		/// Changes the effect of using an item with axe power, melee damage, no projectile, and a swing usestyle to the intended alternative effect
+		/// </summary>
+		/// <param name="item"></param>
+		/// <param name="player"></param>
+		/// <returns></returns>
+		private bool OverrideAxeEffects(Item item, Player player)
```
Comment (TripleFate) [inline Content/Items/Misc/Accessories.AxeBook.cs:60]: This probably shouldn't entirely override axe functionality. I would expect to still be able to chop trees down with this equipped

File: Content/Items/Misc/Accessories.AxeBook.cs [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+
+namespace StarlightRiver.Content.Items.Misc
+{
+	internal class AxeBook : SmartAccessory
+	{
+		public int comboState;
+
+		public override string Texture => AssetDirectory.MiscItem + "AxeBook";
+
+		public AxeBook() : base("Tiger Technique", "Allows execution of combos with axes\nThe final strike will rend enemy armor\nRight click to throw your axe") { }
```
Comment (TripleFate) [inline Content/Items/Misc/Accessories.AxeBook.cs:18]: We really need some kind of expanded tooltips,. When I see this I wonder if this rend effect is good. I'd want to know things like if this debuff stacks, how much it reduces the armor by (which we should probably stay consistent with vanilla and call it defense anyway) how long it lasts. Vanilla is also really bad about this, requiring you to go to the wiki to actually determine if an item is good but we could be better

File: Content/Items/Misc/Accessories.AxeBook.cs [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+						modProj.comboState = comboState;
+					}
+
+					float pitch = -0.3f;
+					pitch += comboState * 0.1f;
+
+					if (pitch >= 1)
+						pitch = 1;
+
+					//Helper.PlayPitched("ChainHit", 1, pitch, player.Center);
```
Comment (TripleFate) [inline Content/Items/Misc/Accessories.AxeBook.cs:117]: remove comment

**Review by TripleFate — CHANGES_REQUESTED** [review]

**Pull request merged**